planner/src/main/java/com/dask/sql/parser/SqlShowModels.java

```diff
@@ -0,0 +1,37 @@
+package com.dask.sql.parser;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.calcite.sql.SqlCall;
+import org.apache.calcite.sql.SqlIdentifier;
+import org.apache.calcite.sql.SqlOperator;
+import org.apache.calcite.sql.SqlWriter;
+import org.apache.calcite.sql.SqlNode;
+
+import org.apache.calcite.sql.parser.SqlParserPos;
+
+public class SqlShowModels extends SqlCall {
+    public SqlIdentifier catalog;
```

Comment:
You do not need those two variables.

Reply:
I will remove this variables , like and catalog

Reply:
Thanks